tests/task/test_eval.py

```diff
@@ -6,14 +6,15 @@
 import os
 
 import pytest
-from fixtures import DataDir, en_sample, ru_sample
+from pathlib import Path
+from tests.fixtures import DataDir, en_sample, ru_sample
 
 en_fake_translated = "\n".join([line.upper() for line in ru_sample.split("\n")])
 ru_fake_translated = "\n".join([line.upper() for line in en_sample.split("\n")])
 
 current_folder = os.path.dirname(os.path.abspath(__file__))
-fixtures_path = os.path.join(current_folder, "fixtures")
-root_path = os.path.abspath(os.path.join(current_folder, ".."))
+fixtures_path = (Path(__file__).resolve().parents[1] / "fixtures").as_posix()
```

Comment:
I wouldn't mix the pathlib and `os.path` utilities here. I would prefer one or the other.

The simplest would be to make this work without pathlib. I generally prefer pathlib, but that does mean changing more lines of code here.